backend/script/mongodb.js

```diff
@@ -26,7 +27,8 @@ db.ex_validator.createIndex({"operator_address": 1}, {"unique": true});
 db.ex_validator.createIndex({"proposer_addr": 1}, {"unique": true,"background": true});
 db.ex_validator.createIndex({"jailed": -1, "status": -1, "voting_power": -1}, {"background": true});
 db.ex_gov_params.createIndex({"key": 1}, {"unique": true});
-db.ex_asset_tokens.createIndex({"id": 1}, {"unique": true, "background": true});
+db.ex_asset_tokens.createIndex({"token_id": 1}, {"unique": true, "background": true});
+db.ex_asset_gateways.createIndex({"identity": 1}, {"unique": true, "background": true});
```

Comment:
ex_asset_gateways unique key should be `moniker`, see detail [https://www.irisnet.org/docs/zh/cli-client/asset/edit-gateway.html](https://www.irisnet.org/docs/zh/cli-client/asset/edit-gateway.html)